include/revng/ABIAnalyses/Generated/DeadRegisterArgumentsOfFunction.h

```diff
@@ -0,0 +1,169 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+// This file has been automatically generated from
+// scripts/monotone-framework-lattice.py, please don't change it
```

Comment:
Submitting for review commits with generated code.
What a brave man. I admire you. 😄